clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForDsn.ts

```diff
@@ -33,14 +34,13 @@ export class CopybookDownloaderForDsn extends ZoweExplorerDownloader {
     _copybookName: CopybookName,
     documentUri: string,
     dsnPath: string | undefined,
+    profile?: string,
```

Comment:
Same here, just make it a mandatory parameter.

Reply:
This function is no longer used (never been needed) the check is already handled at: https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/d6e51dfe38340168334e43bf17ac940a24d5e282/clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts#L358